src/utils/weather-utils.ts

```diff
@@ -0,0 +1,7 @@
+import { WeatherType } from "#enums/weather-type";
+
+//#region Const Exports
+
+export const SUNNY_WEATHER_TYPES = Object.freeze<WeatherType[]>([WeatherType.SUNNY, WeatherType.HARSH_SUN]);
```

Comment:
```suggestion
export const SUNNY_WEATHER_TYPES = Object.freeze([WeatherType.SUNNY, WeatherType.HARSH_SUN]);
```
You don't need to specify the type for arrays of primitives:
![image](https://github.com/user-attachments/assets/f0bb527f-3810-465b-8300-cdc635ba175e)

Reply:
Also this should probably just be in a `constants/` file. Maybe there could be a `constants/weather.ts` file for this and the other one you added?